modules/atmosphere/rendering/renderableatmosphere.cpp

```diff
@@ -132,6 +136,25 @@ namespace {
         "Enable/Disables hard shadows through the atmosphere"
     };
 
+    constexpr openspace::properties::Property::PropertyInfo AtmosphereDimmingHeightInfo ={
+        "AtmosphereDimmingHeight",
+        "Atmosphere Dimming Height",
+        "Percentage of the atmosphere where other objects, such as the stars, are faded."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo SunsetAngleStartInfo = {
```

Comment:
Would there be any drawback from combining this into one Vec2Property with the `Property::ViewOption::MinMax` view?

Reply:
No, I thought of that but didn't find the property - good call!